src/Lexer.php

```diff
@@ -788,18 +788,21 @@ protected function scanTableRow()
 
     private function parseTableCell(string $cell): string
     {
+        $trimmedCell = preg_replace('/^[ \\t\\n\\x0B\\f\\r\\x85\\xA0]++|[ \\t\\n\\x0B\\f\\r\\x85\\xA0]++$/u', '', $cell);
```

Comment:
I noticed this is slightly different to cucumber/gherkin, they [call their `ltrimKeepNewLines` and `rtrimKeepNewLines` utils](https://github.com/cucumber/gherkin/blob/6b9e4c16c0a659396d86c60d8d49a5362c3675f2/php/src/StringGherkinLine.php#L90-L92) which use a regex the same as yours but without the `\n` character.

I think that's OK though - since we're parsing content from a single line of the file, there can't actually be any literal newlines in the string at this point, so there's no particular reason why gherkin need to use `..trimKeepNewLines` over `..trim` here.

But just thought worth sense checking that assumption.